pkg/attester/attester.go

```diff
@@ -152,3 +164,43 @@ func (oi *occurrenceInput) addOccurrence(occurrence *grafeas.Occurrence) error {
 	oi.Occurrences = append(oi.Occurrences, occurrenceAsMap)
 	return nil
 }
+
+func NewList() *List {
+	return &List{
+		attesters: map[string]Attester{},
+	}
+}
+
+type List struct {
+	attesters map[string]Attester
+}
+
+func (al *List) Get(name string) (Attester, error) {
+	if attester, exists := al.attesters[name]; !exists {
+		return nil, fmt.Errorf("Attester \"%s\" not found", name)
```

Comment:
Lowercase on the error message?
```suggestion
		return nil, fmt.Errorf("attester \"%s\" not found", name)
```